website/search/elastic_search.py

```diff
@@ -454,6 +454,10 @@ def update_file(file_, index=None, delete=False):
     node_url = '/{node_id}/'.format(node_id=file_.node._id)
 
     parent_url = '/{}/'.format(file_.node.parent_node._id) if file_.node.parent_node else None,
+    parent_title = file_.node.parent_node.title if file_.node.parent_node else None,
```

Comment:
Just noticed that there's a trailing comma after the parent_url assignment as well. Would you mind removing that?